products/bigquery/terraform.yaml

```diff
@@ -64,6 +64,29 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         exclude: true
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       constants: templates/terraform/constants/bigquery_dataset.go.erb
+  DatasetAccess: !ruby/object:Overrides::Terraform::ResourceOverride
+    exclude_import: true # no unique way to specify
```

Comment:
we will probably need a mutex on the dataset so multiple accesses edits aren't attempted at the same time

also, might be a good idea to add a test that adds two accesses (which tends to catch this)